### ISSUE TITLE
Android - Steps Screen - Missing Heading Levels

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#30]
Each of the four items/sections under the “Step” heading level one (h1) should be a heading level two (h2).

The four items I'm referring to are the ones listed below:

- Learning Objectives
- Key Terms
- Topics
- Further Resources

**Comment by KyleLivingstonDOL** [comment]
It looks like this is complete. UAT will begin June 10.

**Comment by jfc3-dol** [comment]
TalkBack is not reading these as a heading but as a button. I'm not sure is TalkBack can have it be both or maybe it's the way the item is coded.

What is your thought on this?

**Comment by LeHaine** [comment]
The "title" of each collapsible was coded to be a button.

**Comment by jfc3-dol** [comment]
I was expecting them to be a heading that acted like a button and open the hidden content.

Here is an accessible example of what I'm talking about from Heydon Pickering using AIRA.

http://heydonworks.com/practical_aria_examples/#progressive-collapsibles

**Comment by KyleLivingstonDOL** [comment]
John, I think we would want to keep it a button since it is only a truncated search result. We would want those who are hard of sight to know that they can indeed click on it rather than it being a heading, only to have Talkback suddenly state "continue reading." I think if we have it as a button, then there is a clear indication that the user can click on the button to be taken to the search results.

Thoughts?
-Kyle

**Comment by jfc3-dol** [comment]
Sound like a plan for now. So I'm going to close the issue.
